config/locales/en.yml

```diff
@@ -493,19 +493,19 @@ en:
   routing_page:
     body_text: "You can send people directly to the question or page you want, based on their previous answer. \nThis means they’ll only see questions that are relevant to them.\n"
     dropdown_default_text: Select a question to start your route from
-    legend_hint_text: A route can only start from a question where people select one item from a list
+    legend_hint_text: A route can only start from a question where people select one item from a list. You can only add one route from each question.
     legend_text: Which question do you want your route to start from?
-    no_remaining_routes: A route can only start from a question where people select one item from a list. You can only add one route per question.
+    no_remaining_routes: A route can only start from a question where people select one item from a list. You can only add one route from each question.
     no_remaining_routes_heading: You have no more questions to start a route from
     routing_requirements_not_met_heading: What you need to create a route
     routing_requirements_not_met_html: |
       <p>Before you can add a route you’ll need:</p>
       <ul class="govuk-list govuk-list--bullet">
-        <li>2 or more questions</li>
+        <li>at least 2 questions</li>
```

Comment:
@aliuk2012, this should be "2 or more questions", not "at least 2 questions". 

Apologies - I'd updated this so that it read "2 or more questions" on one section of the Mural but not on the one you were working from for this ticket!

Reply:
I've edited that first bullet point under "Before you can add a route you'll need [...]"